src/kata6.joinNames.js

```diff
@@ -1,3 +1,14 @@
-const joinNames = (namesObj) => {};
+const joinNames = (namesObj) => {
+  const list = namesObj.map((object) => object.name);
+  const last = list.pop();
+  return `${list.join(", ")} & ${last}`;
+  //   const list = namesObj.reduce((acc, curr, index) => {
+//     if (index < namesObj.length - 1) {
+//       return `${acc}, ${curr.name}`;
+//     } 
+//     return `${acc} & ${curr.name}`;
+//   }, namesObj[0].name);
+//   return list;
```

Comment:
Nice solution! All this commented out code should be deleted though, its just cluttering up your already working answer